apps/coscrad-frontend/src/components/resources/factories/aggregate-string-summarizer-factory.tsx

```diff
@@ -22,8 +22,7 @@ export const aggregateStringSummarizerFactory = (
         };
 
     if (aggregateType === AggregateType.mediaItem)
-        return ({ title, titleEnglish }: IMediaItemViewModel): string =>
-            formatBilingualText(title, titleEnglish);
+        return ({ name }: IMediaItemViewModel): string => findOriginalTextItem(name).text;
```

Comment:
This might be able to be the logic for all view models now that they all have names.

Reply:
Nice!